src/main/java/step4/dto/RacingCars.java

```diff
@@ -0,0 +1,30 @@
+package step4.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingCars {
+
+    private List<Car> racingCars;
+
+    public RacingCars(List<String> carsName) {
+        racingCars = new ArrayList<>();
+
+        carsName.forEach(name -> {
+            racingCars.add(new Car(new CarName(name)));
+        });
+    }
+
+    public void moveRacingCars(RunStrategy runStrategy) {
+        racingCars.forEach(car -> {
+            car.run(runStrategy);
+        });
+    }
+
+    public List<Car> currentCars() {
+        return racingCars.stream()
+                .map(car -> new Car(car))
```

Comment:
```suggestion
                .map(Car::new)
```

혹은

```suggestion
                .map(car::deepCopy)
```

이런식으로 할 수 있겠죠. `method reference` 를 애용해주세요 ㅎㅎ